src/test/java/com/hongliangjie/fugue/serialization/FeatureTest.java

```diff
@@ -29,4 +29,34 @@ public void testGetFeatureValue() throws Exception {
         f.setFeatureValue(1.0);
         assertEquals("Testing FeatureValue", true, Math.abs(f.getFeatureValue() - 1.0) < 1e-10);
     }
+
+    @Test
+    public void testGetFeatureType1() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/hongliangjie/Fugue/file/2176917789/issues/source?bid=3162994&fileBranchId=3163615#l34)